lib/helpers/iterate-parents.js

```diff
@@ -0,0 +1,8 @@
+function* iterateParents(path) {
```

Comment:
I think we can remove this now that https://github.com/glimmerjs/glimmer-vm/pull/1024 landed and is released.

Reply:
This utility needs to be removed, and update the internals of the rule to use `path.parents` directly